000 - Introduction.md

```diff
@@ -1,34 +1,34 @@
 
 # Introduction
 
-Implementing a containerized environment is changing a lot not only how deployments are done but it also has a huge impact on a system and networking level and how hardware and network resources are being used.
+Implementing a containerized environment changes a lot not. It doesn't only affect deployments and the way they are done, but also has a huge impact on both system and networking levels, as well as hardware and network resource usage.
```

Comment:
The part with `changes a lot not.` looks mistaken to me. Did you forget to drop `not` here?